invenio_rdm_migrator/streams/files/table_generator.py

```diff
@@ -32,3 +36,7 @@ def _generate_rows(self, data, **kwargs):
         yield FilesInstance(**file)
         yield FilesBucket(**bucket)
         yield FilesObjectVersion(**object_version)
+
+
+FilesBucketTableGenerator = partial(MemorySingleTableGenerator, table=FilesBucket)
```

Comment:
another naming problem, it can not be inferred if it uses CSV or in memory. However, adding to the name would make it even longer. Considering dropping _TableGenerator_ since it's already in the package where it is imported from.